src/scielo/bin/xml/prodtools/utils/logging_config.py

```diff
@@ -1,7 +1,6 @@
 
 LOGGING_CONFIG = {
     'version': 1,
-    'disable_existing_loggers': True,
```

Comment:
Sabe a sugestão que dei para desabilitar essa linha? Então, parece que o valor padrão é `True` assim ao remover a linha nós a mantemos ligada. Talvez isso nem seja um problema mas eu pude reparar diferença no log com essa linha removida e com o valor setado em `False`.

Se o valor for setado como `False` nós temos mais informações no fim da execução (https://gist.github.com/joffilyfe/aa470eb6dc65d6e7d4c7393e66ad7a2c). Parece que são os logs do `packtools`.

Reply:
@robertatakenaka uma outra coisa que eu venho notando neste arquivo de configurações de log é que esse formato `brief` esconde a data do log, isso é proposital?

Reply:
> Sabe a sugestão que dei para desabilitar essa linha? Então, parece que o valor padrão é `True` assim ao remover a linha nós a mantemos ligada.

Sim, o valor padrão é `True`, segundo a documentação.

> Talvez isso nem seja um problema mas eu pude reparar diferença no log com essa linha removida e com o valor setado em `False`.
> 
> Se o valor for setado como `False` nós temos mais informações no fim da execução (https://gist.github.com/joffilyfe/aa470eb6dc65d6e7d4c7393e66ad7a2c). Parece que são os logs do `packtools`.

Pois é, tem diferença sim. Segundo a documentação, somente os loggers com perfil `root` seria expelidos sem este parâmetro:

> _disable_existing_loggers: If specified as False, loggers which exist when this call is made are left enabled. The default is True because this enables old behaviour in a backward-compatible way. This behaviour is to disable any existing non-root loggers unless they or their ancestors are explicitly named in the logging configuration._

Portanto, vou retornar o parâmetro por acreditar que este não seja o comportamento desejado.

Reply:
> @robertatakenaka uma outra coisa que eu venho notando neste arquivo de configurações de log é que esse formato `brief` esconde a data do log, isso é proposital?

Este _formatter_ é usado no log padrão, que emite os logs em tela. Eu adicionaria a sua pergunta se é desejado que os logs em console devem ser emitidos sem data e hora.

Reply:
era porque eu achava que funcionaria como o print, senão é melhor manter completo com data/hora...